install-php-extensions

```diff
@@ -926,6 +926,12 @@ buildRequiredPackageLists() {
 			xlswriter@debian)
 				buildRequiredPackageLists_volatile="$buildRequiredPackageLists_volatile zlib1g-dev"
 				;;
+			xmldiff@alpine)
+				buildRequiredPackageLists_volatile="$buildRequiredPackageLists_volatile libxml2-dev"
```

Comment:
It seems we also need `libstdc++ ` (persistent)

Reply:
probably : i'm sorry but I dind't run the script locally and test it first (it's very strange behavior)

Reply:
That's why we have that GitHub Action 😉

Reply:
Finally found it how to test it locally.